### PR TITLE
Snaxi Taxi

### LINKED ISSUE #25327: Snowmap: Winter coat storage access requirements are too high on the south sector.
[snowmap]
#### Description of issue
The winter coat storage on the south station section are behind an engineering access airlock whle others are behind open access ones.
A sec officer arrested me for 3 minutes for rotating the windows to get a coat. Either the map needs to be changed or space law does because this made me sad.

### PR DESCRIPTION
**Pull request by Kurfursten**
This is a huge PR and WiP, so brief summary:
image
climate
* NEW! Climate computers let you see current temperature and snowfall, intended for use with...
* Brought back taxis to snaxi. They go to the new roid outpost, so you have an alternative to walking.
_It is intended in the upcoming weather update, heavy storms will interfere with these taxis_
* New decals: Med, Sci, Sec, Cargo, Bar, HoP, and TAXI sign. Never get lost again.
* New taxi docks on each of the 3 stations
* Outpost now sports the pod bay and ghetto bar. Addressing worries about a lack of bar on the surface, I finally fulfilled my promise by adding an ice cave bar in the enclosed forest. Old location of the ghetto bar is now a construction site.
* Outpost botany is now revamped entirely for your pleasure
* fixes #25327

**CURRENT**
Untitled

**OUTDATED PICS**
snaxi 01
snaxi 02
snaxi 03
snaxi 04
snaxi 05
snaxi 06
snaxi 07
snaxi 08

I know the pipes, lights, etc. are not done, but it was staggeringly huge already so I thought I'd put it up for feedback and make sure it's not going to get a lot of hostility.

🆑 
* rscadd: We put the taxi back in Snaxi.
* bugfix: On Snaxi, the wintercoats on south station no longer require engineering access.

### CODE REVIEW

**Comment by plosky1** [comment]
Noice.

**Comment by Kurfursten** [comment]
Finally done. Tested it pretty thoroughly, too.

**Comment by N3X15** [comment]
The existing letter floor decals were made with the Bebas Neue font, if that helps any.

File: code/game/machinery/computer/climate.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	add_hiddenprint(user)
+	return attack_hand(user)
+
+/obj/machinery/computer/climate/attack_hand(var/mob/user as mob)
+	if(..())
+		return
+	user.set_machine(src)
+	var/dat = list()
+	dat += "<center>"
+	dat += "<div class='modal'><div class='modal-content'><div class='line'><b>Weather Report</b></div><br>"
```
Comment (N3X15) [inline code/game/machinery/computer/climate.dm]: Screenshot please.

File: code/game/machinery/computer/climate.dm [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	dat += "<b>Temperature:</b> <div class='line'>[reported_temp] Celcius</div>"
+	dat += "<b>Snowfall:</b> <div class='line'>[reported_snow] </div></div></div></center>"
```
Comment (N3X15) [inline code/game/machinery/computer/climate.dm:31]: Screenshot of this display, please.

**Review by N3X15 — COMMENTED** [review]
Also please update the screenshots you said were outdated.  A map change like this needs some review.

Pipe/power views would also be helpful.

**Comment by Kurfursten** [comment]
updated OP with the pipe/power pic

the rest have had some minor changes (for example, the docks at the medsci and engcargo stations had to be reshuffled slightly) and an O2 locker removed at the secmine station, however no major changes took place except on the outpost so I only posted that one new pic

**Pull request merged**